ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -352,12 +354,10 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{ na
     private filterTreeNode(node: ResourceTreeNode, filter: {kind: string[], health: string[], sync: string[]}): boolean {
         const syncStatuses = filter.sync.map((item) => item === 'OutOfSync' ? ['OutOfSync', 'Unknown'] : [item] ).reduce(
             (first, second) => first.concat(second), []);
-        const healthStatuses = filter.health.map((item) => item === 'Unhealthy' ? ['Unknown', 'Progressing', 'Degraded', 'Missing'] : [item] ).reduce(
```

Comment:
This is now unnecessary since all options of `node.root.health` are now exposed to the UI.